### PR TITLE
Update ptBR.lua

### PR DESCRIPTION
**Pull request by tuliobaruk**
Updating Aberrus ptBR locales.

### CODE REVIEW

**Comment by funkydude** [comment]
Hello do you plan on also updating the follow files?
https://github.com/BigWigsMods/BigWigs/blob/master/Core/Locales/common.ptBR.lua
https://github.com/BigWigsMods/BigWigs/blob/master/Locales/ptBR.lua

File: Aberrus/Locales/ptBR.lua [hunk 1]
```diff
@@ -1,106 +1,106 @@
...
+	L.temporal_anomaly_knocked = "Orbe de Cura Empurrado"
 end
 
 L = BigWigs:NewBossLocale("Assault of the Zaqali", "ptBR")
 if L then
 	-- These are in-game emotes and need to match the text shown in-game
 	-- You should also replace the comment (--) with the full emote as it shows in-game
-	--L.zaqali_aide_north_emote_trigger = "northern battlement" -- Commanders ascend the northern battlement!
-	--L.zaqali_aide_south_emote_trigger = "southern battlement" -- Commanders ascend the southern battlement!
+	L.zaqali_aide_north_emote_trigger = "Reforço Norte" -- Commanders ascend the northern battlement!
```
Comment (funkydude) [inline Aberrus/Locales/ptBR.lua:41]: We would like the comments here (--) replaced with the full ptBR text copy paste from the in-game emote.

**Comment by tuliobaruk** [comment]
Fixed emote text

**Comment by tuliobaruk** [comment]
> Hello do you plan on also updating the following files?
> https://github.com/BigWigsMods/BigWigs/blob/master/Core/Locales/common.ptBR.lua
> https://github.com/BigWigsMods/BigWigs/blob/master/Locales/ptBR.lua

I will check the files and translate what is needed 👍

**Comment by tuliobaruk** [comment]
Done

**Comment by funkydude** [comment]
Excellent work, thank you!

**Pull request merged**